docs/sources/reference/components/otelcol/otelcol.exporter.file.md

```diff
@@ -0,0 +1,246 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.exporter.file/
+description: Learn about otelcol.exporter.file
+labels:
+  stage: general-availability
+  products:
+    - oss
+  tags:
+    - text: Community
+      tooltip: This component is developed, maintained, and supported by the Alloy user community.
```

Comment:
```suggestion
```

I don't think this needs to be a community component, as it doesn't require non-grafana vendor environments to validate/test.